lib/chitter_peeps.rb

```diff
@@ -0,0 +1,30 @@
+require 'pg'
+
+class Chitter_peeps
+  attr_reader :peep
+
+  def initialize(peep)
+    @peep = peep
+  end
+
+  def self.all
+    if ENV['RACK_ENV'] == 'test'
+      connection = PG.connect(dbname: 'chitter_challenge_test')
+    else
+      connection = PG.connect(dbname: 'chitter_challenge')
+    end
+    result = connection.exec("SELECT * FROM chitter")
+    result.map {|peep| Chitter_peeps.new(peep['peeps'])}
+  end
+
+
+  def self.create(peep:)
+    if ENV['RACK_ENV'] == 'test'
```

Comment:
Nice and clear 👍 
My next steps would have been to separate the PG.connect lines into another method to DRY out the code.